pkg/virt-api/webhooks/mutating-webhook/mutators/preset.go

```diff
@@ -66,50 +66,50 @@ func filterPresets(list []kubev1.VirtualMachineInstancePreset, vmi *kubev1.Virtu
 	return matchingPresets, nil
 }
 
-func checkMergeConflicts(presetSpec *kubev1.DomainSpec, vmiSpec *kubev1.DomainSpec) error {
+func checkDomainMergeConflicts(presetSpec *kubev1.DomainSpec, vmiSpec *kubev1.DomainSpec) error {
```

Comment:
As already suggested I'd drop all of the preset changes given the deprecation.

Reply:
I personally don't mind landing this even if the overall feature is deprecated, I however wouldn't like to see it advertised in the releasenotes or user-guide so we can continue to drive users towards instance types etc.

Reply:
Sure, I can remove it from release notes.